dashboard/pages/5_Word_Clouds.py

```diff
@@ -58,6 +60,16 @@ def get_all_article_content():
     return fox_news_articles, democracy_now_articles
 
 
+w_tokenizer = WhitespaceTokenizer()
```

Comment:
Could you format the name in all-caps since it's a global variable?

Reply:
Yup, will do that now!